deploy/overlays/private/kustomization.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kustomization for "private" mixer running on GCP `datcom-mixer-private` project.
+# - Adds "private" suffix to all the resources.
+# - Use replica of 5.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -private
+
+resources:
+  - ../../datacommons
+
+configMapGenerator:
+  - name: mixer-configmap
+    behavior: create
+    namespace: mixer
+    literals:
+      - mixerProject=datcom-mixer-private
+      - serviceName=mixer.endpoints.datcom-mixer-private.cloud.goog
+
+patchesStrategicMerge:
+  - |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: mixer-grpc
+    spec:
+      replicas: 5
+      strategy:
+        type: RollingUpdate
+        rollingUpdate:
+          maxSurge: 3
+          maxUnavailable: 75%
+      template:
+        spec:
+          containers:
+            - name: mixer
+              args:
+                - --mixer_project=$(MIXER_PROJECT)
+                - --use_gcs_data=true
+                - --gcs_bucket=datcom-public
+                - --gcs_object_prefix=iea
+                - --use_bigquery=false
```

Comment:
if use_base_bt=false, does it mean this won't do merge?

Reply:
Yes, I want to start with data from in-memory cache only, and test out other APIs